core/src/main/resources/hudson/model/View/sidepanel_pl.properties

```diff
@@ -22,7 +22,7 @@
 
 Build\ History=Historia zada\u0144
 Check\ File\ Fingerprint=Sprawd\u017A Odcisk Palca Pliku
-Delete\ View=Usu\u0144 Widok
+Delete\ View=Usu\u0144 widok
```

Comment:
Not sure I understand that answer.

The question @daniel-beck is asking is IIUC: in English, the [rule is that (almost) every words must be capitalized](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html). 
From your translations, it looks like there's the same rule in Polish? Is that the case?

Thanks